packages/flutter_tools/lib/src/migrate/migrate_compute.dart

```diff
@@ -0,0 +1,620 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../base/logger.dart';
+import '../base/terminal.dart';
+import '../flutter_project_metadata.dart';
+import '../project.dart';
+import '../version.dart';
+import 'custom_merge.dart';
+import 'migrate_result.dart';
+import 'migrate_utils.dart';
+
+// This defines files and directories that should be skipped regardless
+// of gitignore and config settings
+const List<String> _skippedFiles = <String>[
+  'lib/main.dart', // Almost always user owned.
```

Comment:
is this needed since `lib` is under `_skippedDirectories`?

Reply:
No, this is probably redundant. I can remove.